rpm/BUILD.bazel

```diff
@@ -325,6 +325,7 @@ rpmtree(
     ],
     symlinks = {
         "/var/run": "../run",
+        "/usr/sbin/iptables": "/usr/sbin/iptables-legacy",
```

Comment:
Is this what iptables post-install script does?

Reply:
Yes:

```
$ sudo alternatives --display iptables
iptables - status is auto.
 link currently points to /usr/sbin/iptables-legacy
/usr/sbin/iptables-legacy - priority 10
 slave ip6tables: /usr/sbin/ip6tables-legacy
 slave iptables-restore: /usr/sbin/iptables-legacy-restore
 slave iptables-save: /usr/sbin/iptables-legacy-save
 slave ip6tables-restore: /usr/sbin/ip6tables-legacy-restore
 slave ip6tables-save: /usr/sbin/ip6tables-legacy-save
Current `best' version is /usr/sbin/iptables-legacy.
```